src/plasma/lib/python/plasma.py

```diff
@@ -224,29 +168,17 @@ def transfer(self, addr, port, object_id):
       port (int): Port number of the plasma instance the object is sent to.
       object_id (str): A string used to identify an object.
     """
-    if not self.has_manager_conn:
-      raise Exception("Not connected to the plasma manager socket")
-    self.client.plasma_transfer(self.plasma_conn, addr, port, make_plasma_id(object_id))
+    return libplasma.transfer(self.conn, object_id, addr, port)
 
   def fetch(self, object_ids):
     """Fetch the object with id object_id from another plasma manager instance.
 
     Args:
       object_id (str): A string used to identify an object.
     """
-    object_id_array = (len(object_ids) * PlasmaID)()
-    for i, object_id in enumerate(object_ids):
-      object_id_array[i] = make_plasma_id(object_id)
-    success_array = (len(object_ids) * ctypes.c_int)()
-    if not self.has_manager_conn:
-      raise Exception("Not connected to the plasma manager socket")
-    self.client.plasma_fetch(self.plasma_conn,
-                             object_id_array._length_,
-                             object_id_array,
-                             success_array);
-    return [bool(success) for success in success_array]
-
-  def wait(self, object_ids, timeout, num_returns):
+    return libplasma.fetch(self.conn, object_ids)
+
+  def wait(self, object_ids, timeout=2**32-1, num_returns=1):
```

Comment:
whitespace, just save a variable `timeout = 2 ** 32 - 1` and then `timeout=timeout`.